carbonforwarderlogging/forwarder_log_observer.py

```diff
@@ -10,6 +11,14 @@
 
 def get_log_observer():
     f = logfile.LogFile("carbon_forwarder.log", log_dir, rotateLength=log_rotate_length, maxRotatedFiles=max_rotated_log_files)
+    observer = FileLogObserver(f)
+    filterer = FilteringLogObserver(observer.emit,
+        [LogLevelFilterPredicate(
+            LogLevel.levelWithName(log_level))])
+    return filterer
+
+def get_json_log_observer():
```

Comment:
It is called this way:
```
    twistd --logger carbonforwarderlogging.forwarder_log_observer.get_json_log_observer blueflood-forward
```
That's the documented way to define custom logging in the twistd world.